code/modules/atmospherics/machinery/portable/portable_pump.dm

```diff
@@ -21,8 +21,9 @@
 /obj/machinery/atmospherics/portable/pump/examine(mob/user)
 	. = ..()
 	. += "<span class='notice'>Invaluable for filling air in a room rapidly after a breach repair. The internal gas container can be filled by \
-			connecting it to a connector port. The pump can pump the air in (sucking) or out (blowing), at a specific target pressure. The powercell inside can be \
-			replaced by using a screwdriver, and then adding a new cell. A tank of gas can also be attached to the air pump.</span>"
+			connecting it to a connector port, you're unable to have [src] both connected, and on at the same time. \
+			[src] can pump the air in (sucking) or out (blowing), at a specific target pressure. \
+			A tank of gas can also be attached to the air pump, alowing you to fill or empty the tank, via the internal one.</span>"
```

Comment:
"alowing".
Also I feel like it flows better to use "it" rather than the name, but that's personal preference.
Comma placement on the last line is a bit awkward, I'd recommend taking out the second one.

Reply:
You have a button in the comment tab that looks like the `request changes`  symbol, that allows you to suggest changes to the code which makes it easier for the author to read!

Reply:
Duly noted! I'll try it next time

Reply:
> "alowing".

fixed, i've also changed some of the `[src]` around.